### DISCUSSION TITLE
Weekly update 2020-12-04

### DISCUSSION THREAD

**Discussion by alexauroradev** [near/nearcore#3701] [EVM]
https://www.youtube.com/watch?v=uP5sRyDiZLM

# Current goal
EVM branch merged into master (one issue occur). EVM feature was released to `Betanet`.

# Weekly update

**Alex**:
* EVM roadmap with Eugene
* EVM partner form
* Cleanup in Github with Arto

**Bo**:
* No major updates

**Arto**:
* CryptoZombies. Handed over it to Frank. https://github.com/near/nearcore/issues/3670
* Anemone. Started the work on it.
* Balancer tests fixed together with Mike. https://github.com/near/nearcore/issues/3665
* Onboarding Frank
* Doc updates. Some PRs that would help to onboard Frank.
* Chainsafe cooperation. Aiden requires some time schedules, need more inputs here.

**Eugene**:
* Major PR is merged
* Plans to work on the benchmarks (ERC20 transfer)

**Frank**:
* Started this week
* Started to work with CryptoZombies

**Mike**:
* Sick leave
* Pet shop updated, all works good, need to be updated once the PR is merged into the NEAR Core (prerequisite: user has NEAR account)
* Updated the info on setting up the NEAR stack locally. This is crucial for the EVM development, it's going to be included in the Pet shop docs.
* Moloch are using Hardhat, starting to dig into how we can enable this

**Discussion**
* Management of the projects: how to measure individual velocities and deal with burn down charts?
  * Arto: JIRA is the choice for the projects of this size
  * Frank: Github is not doing this
  * Alex AI: try ZenHub and report next week
* Move to Discord?
  * Functionality is OK
  * Many things can be structured as GitHub issues, so let's do this
  * Eugene created EVM channel
* Structured error messages from `nearcore` <- this hits us a lot
  * Max: status - it's implemented, but JS adapters are not there. Follow up with @bowenwang1996 @frol.

**Comment by frol** [comment]
> Structured error messages from nearcore <- this hits us a lot
> Max: status - it's implemented, but JS adapters are not there. 

@djsatok @nearmax The implementation stuck in PR https://github.com/near/nearcore/pull/3204, and neither I nor @bowenwang1996 have the capacity to update it and merge the first stage #3518. Next, it will require a backward-compatible implementation in JSON RPC #3517, and then we will need to design and implement the structured errors as of #2976. It is quite far from getting any progress any time soon.

**Reply by alexauroradev** [reply]
@frol @nearmax, can this be prioritised for mainnet guild for, e.g. Q1?

**Reply by frol** [reply]
Yes, we should have the bandwidth for that in Q1.
